structure-of-a-contract.rst

```diff
@@ -2,63 +2,58 @@
 
 .. _contract_structure:
 
-***********************
-Structure of a Contract
-***********************
+*************************
+Estructura de un Contrato
+*************************
 
-Contracts in Solidity are similar to classes in object-oriented languages.
-Each contract can contain declarations of :ref:`structure-state-variables`, :ref:`structure-functions`,
+Los Contratos en Solidity son similares a las clases de los lenguajes orientados a objeto.
+Cualquier contrato puede contener declaraciones del tipo :ref:`structure-state-variables`, :ref:`structure-functions`,
 :ref:`structure-function-modifiers`, :ref:`structure-events`, :ref:`structure-structs-types` and :ref:`structure-enum-types`.
-Furthermore, contracts can inherit from other contracts.
+Además, los contratos pueden heredar de otros contratos.
 
 .. _structure-state-variables:
 
-State Variables
-===============
+Variables de Estado
+===================
 
-State variables are values which are permanently stored in contract storage.
+Las Variables de Estado son valores que están permanentemente almacenados en una parte del contrato conocida como almacen del contrato.
 
 ::
 
   pragma solidity ^0.4.0;
 
   contract SimpleStorage {
-      uint storedData; // State variable
+      uint storedData; // Variable de estado
       // ...
   }
 
-See the :ref:`types` section for valid state variable types and
-:ref:`visibility-and-getters` for possible choices for
-visibility.
+Véase la sección :ref:`types` para conocer los diferentes tipos válidos de variables de estado y :ref:`visibility-and-getters` para conocer las distintas posibilidades de visibilidad que pueden tener las variables de estado.
 
 .. _structure-functions:
 
-Functions
+Funciones
 =========
 
-Functions are the executable units of code within a contract.
+Las Funciones son las unidades ejecutables del código dentro de un contrato.
 
 ::
 
   pragma solidity ^0.4.0;
 
   contract SimpleAuction {
-      function bid() payable { // Function
+      function bid() payable { // Función
           // ...
       }
   }
 
-:ref:`function-calls` can happen internally or externally
-and have different levels of visibility (:ref:`visibility-and-getters`)
-towards other contracts.
+Las llamadas a las funciones (:ref:`function-calls`) pueden ocurrir dentro o fuera de la misma. Una función puede tener varios niveles de visibilidad (:ref:`visibility-and-getters`) con respecto a otros contratos.
```

Comment:
You can use "Las :ref:`llamadas a las funciones <function-calls>`", this will be turned into something like `Las <a href="url_to_function_calls">llamadas a las funciones</a>`.